fastlane/lib/fastlane/actions/carthage.rb

```diff
@@ -31,6 +36,10 @@ def self.available_platforms
       end
 
       def self.available_options
+        default_options + options
```

Comment:
I assume you're talking about a rubocop style error? If so so you can disable the bot for this method using
Run `rubocop -D` to get the cop name and add
```
# rubocop:disable [bot name here]
def ...
# rubocop:enable [bot name here]
```